<launch>

      <node pkg="armor" 
            type="execute"
            name="armor_service" 
            args="it.emarolab.armor.ARMORMainService"
            launch-prefix="xterm -e"
      > </node>

      <node pkg="exprob_robot_patrolling"
            type="joint_pose_controller.py"
            name="joint_pose_controller" 
            output = "screen"
      > </node>
      
      <!--
      <node pkg = "exprob_robot_patrolling"  
            type = "world_generator.py"
            name = "world_generator"
            output = "screen"
      > </node>
      
      
      <node pkg = "exprob_robot_patrolling"  
            type = "fsm_behaviour.py" 
            name = "fsm_behaviour"
            output = "screen"
      > </node>
    
      <node pkg = "exprob_robot_patrolling"  
            type = "battery.py"
            name = "battery_node"
            launch-prefix="xterm -e"
       > </node>-->

</launch>
